Solutions/Corvus.Azure.Storage.Tenancy/Corvus/Azure/Storage/Tenancy/Internal/TenantCloudBlobContainerFactory.cs

```diff
@@ -214,11 +214,9 @@ protected async Task<CloudBlobContainer> CreateCloudBlobContainerInstanceAsync(I
         private static string HashAndEncodeContainerName(string containerName)
         {
             byte[] byteContents = Encoding.UTF8.GetBytes(containerName);
-            using (SHA1 hash = new SHA1CryptoServiceProvider())
-            {
-                byte[] hashedBytes = hash.ComputeHash(byteContents);
-                return TenantExtensions.ByteArrayToHexViaLookup32(hashedBytes);
-            }
+            using SHA1 hash = new SHA1CryptoServiceProvider();
```

Comment:
Is there a particular reason for the variabled type here to be diferent from the instance type?

Reply:
This is the "use the simplest base type" refactoring that is enabled by default. 
However, our policy is to use `var` in this case, so I am switching to that.